README.md

```diff
@@ -60,9 +60,14 @@ Append ```?coverage=true``` to the HTML report URL to enable coverage. This opti
 
 You can add this as part of the ```ember test``` command for a single run:
 
+*Pre ember-cli 1.13.13*
 ```bash
 ember test --test-page='tests/index.html?coverage=true'
 ```
+*Post ember-cli 1.13.13*
+```bash
+ember test --test_page=tests/index.html?coverage=true
```

Comment:
Whew, that's a tight distinction - we should consider adding our own commands (`ember coverage`?) to insulate users from this